### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by hbcarlos**
Updated the readme.

- Updated the binder badge.
- Added screencast for the different extensions

Seams like binder is using JupyterLab 2.2, maybe we have to add jupyterlab=3 as a dependency in the binder configuration file.

### CODE REVIEW

**Comment by jtpio** [comment]
> Seams like binder is using JupyterLab 2.2, maybe we have to add jupyterlab=3 as a dependency in the binder configuration file.

Good catch. Do you want to add it to `environment.yml`?

**Comment by hbcarlos** [comment]
Yes, I'm gonna merge master in my branch and commit

File: README.md [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+![voila-gridstack](assets/voila.png)
+
 [![Join the Gitter Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/QuantStack/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Notebook extension
+
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/voila-dashboards/voila-gridstack/stable?urlpath=/voila/tree/examples/scotch_dashboard.ipynb)
 
-A gridstack-based template for [Voilà](https://github.com/voila-dashboards/voila).
+JupyterLab-gridstack
```
Comment (jtpio) [inline README.md:12]: How about:

```suggestion
JupyterLab extension
```

So it's similar to "Notebook extension" above.

**Comment by jtpio** [comment]
Small nit-pick: would it be possible to resize the voila logo down a little bit?

image

**Comment by hbcarlos** [comment]
Captura de pantalla 2021-01-07 a las 15 51 22

**Review by jtpio — APPROVED** [review]
Nice, thanks!

**Pull request merged**